examples/stable/postgresql-ha/hook-blueprint/README.md

```diff
@@ -0,0 +1,171 @@
+
+## Installing the Chart
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm repo update
+
+# If you are using helm v3
+$ kubectl create ns postgres-ha-test
+$ helm install my-release --namespace postgres-ha-test bitnami/postgresql-ha
+
+# If you are using helm v2
+$ helm install --name my-release --namespace postgres-ha-test bitnami/postgresql-ha
+```
+
+The command deploys PostgreSQL HA on the Kubernetes cluster in the default configuration.
+
+> **Tip**: List all releases using `helm list`
+
+In case, if you don't have `Kanister` installed already, you can use following commands to do that.
+Add Kanister Helm repository and install Kanister operator
+```bash
+$ helm repo add kanister https://charts.kanister.io
+$ helm install --name kanister --namespace kasten-io kanister/kanister-operator --set image.tag=0.71.0
+
+```
+
+### Create Blueprint
+Create Blueprint in the same namespace as the controller
+
+```bash
+$ kubectl create -f ./postgres-ha-hook-blueprint.yaml -n kasten-io
+```
+
+### After Restore of the application
+Lets assume, restore is done in a namespace `postgres-ha-test2`
+
+The pod ``my-release-postgresql-ha-postgresql-1`` in ``postgres-ha-test2`` goes into ``CrashLoopBackoff`` because of wrong connection information
+
+Check the primary/standby connection information in rpmgr database 
+
+```
+Log into the primary node of postgres-ha application and get shell access
+$ kubectl exec -ti my-release-postgresql-ha-postgresql-0 -n postgres-ha-test2 -- bash
+
+I have no name!@my-release-postgresql-ha-postgresql-0:/$ PGPASSWORD=${POSTGRES_PASSWORD} psql -U $POSTGRES_USER
+psql (11.14)
+Type "help" for help.
+
+postgres=# \c repmgr
+You are now connected to database "repmgr" as user "postgres".
+
+repmgr=# select conninfo from repmgr.nodes;
+                                                                                               conninfo                                       
+                                                         
+----------------------------------------------------------------------------------------------------------------------------------------------
+---------------------------------------------------------
+ user=repmgr password=ktA5BvUN4a host=my-release-postgresql-ha-postgresql-1.my-release-postgresql-ha-postgresql-headless.postgres-ha-test.svc.
+cluster.local dbname=repmgr port=5432 connect_timeout=5
+ user=repmgr password=ktA5BvUN4a host= my-release-postgresql-ha-postgresql-0.my-release-postgresql-ha-postgresql-headless.postgres-ha-test.svc
+.cluster.local dbname=repmgr port=5432 connect_timeout=5
+
+```
+
+The conninfo column is pointing to source namespace ``postgres-ha-test``. Hence we need to create an actionset on statefulset in namespace ``postgres-ha-test2``. An easy way to do this is to leverage kanctl, a command-line tool that helps create ActionSets that depend on other ActionSets:
```

Comment:
```suggestion
The `conninfo` column is pointing to source namespace ``postgres-ha-test``. Hence we need to create an actionset on statefulset in namespace ``postgres-ha-test2``. An easy way to do this is to leverage `kanctl`, a command-line tool that helps create ActionSets that depend on other ActionSets:
```

Reply:
`Hence we need to create an actionset on statefulset in namespace ``postgres-ha-test2``.`, we should also try to specify why do we need to create the actionset.